internal/app/networkd/main.go

```diff
@@ -2,66 +2,37 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-package main
+package networkd
```

Comment:
Should we rename this file now that it does not contain the "normal" `main()` function?

Should we also move this file away from `internal/app/` according to the project layout document?

Reply:
Should we also update components documentation?

Reply:
> Should we rename this file now that it does not contain the "normal" `main()` function?

I would rather keep it in `main.go` for now, as it's still an entrypoint to the service.

Reply:
> Should we also update components documentation?

From the user point of view, nothing has changed, as still service `networkd` exists, it's visible in `talosctl services`, `talosctl logs networkd` works, etc. It just changes the way we run it, but it should be transparent to the user.

Reply:
> Should we also move this file away from `internal/app/` according to the project layout document?

same answer, nothing has changed so far, it's still a service and kind of same service. 

networkd itself is supposed to be removed any time soon once we make it COSIfied, and then this directory will go away and docs will be updated